setup.cfg

```diff
@@ -0,0 +1,21 @@
+[options.extras_require]
```

Comment:
what's that for?

Reply:
Same as the (removed) extras_require in the `setup.py`. it allows people to run `pip install pythran[deps]` (as before, install pythran + numpy) or `pip install -e .[dev]` as used in the devguide.